assets/components/molecules/basic-page-teaser/basic-page-teaser.scss

```diff
@@ -3,53 +3,141 @@
 .basic-page-teaser {
   position: relative;
 
+  @extend .col-12;
```

Comment:
`@extend` are great but dangerous, you never know all the stuff that's behind and it can produce a lot of redondant CSS. You can either use the `.col-12` on your component's HTML or add manually what interest you (f.ex. `width: 100; padding: 0 $gutter-width / 2`)